src/commands/moderation/blacklist.js

```diff
@@ -4,7 +4,7 @@ module.exports = class BlacklistCommand extends Command {
   constructor (client) {
     super({
       name: 'blacklist',
-      category: 'developers',
+      category: 'moderation',
```

Comment:
this should stay in the developers category since it's only usable by developers (see the requirements property)